ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthProtocolManager.java

```diff
@@ -412,35 +413,30 @@ private void handleStatusMessage(final EthPeer peer, final Message message) {
     try {
       if (!status.networkId().equals(networkId)) {
         LOG.atDebug()
-            .setMessage("Mismatched network id: {}, EthPeer {}...")
+            .setMessage("Mismatched network id: {}, peer {}")
             .addArgument(status.networkId())
-            .addArgument(peer.getShortNodeId())
-            .log();
-        LOG.atTrace()
-            .setMessage("Mismatched network id: {}, EthPeer {}")
-            .addArgument(status.networkId())
-            .addArgument(peer)
+            .addArgument(LOG.isTraceEnabled() ? peer : peer.getShortNodeId())
             .log();
         peer.disconnect(DisconnectReason.SUBPROTOCOL_TRIGGERED);
       } else if (!forkIdManager.peerCheck(forkId) && status.protocolVersion() > 63) {
         LOG.debug(
             "{} has matching network id ({}), but non-matching fork id: {}",
-            peer,
+            LOG.isTraceEnabled() ? peer : peer.getShortNodeId(),
```

Comment:
maybe make this a method so it can be used consistently in all logging from this class?